examples/text-classification/README.md

```diff
@@ -155,3 +155,59 @@ python run_glue.py \
   --use_hpu_graphs_for_inference \
   --bf16
 ```
+
+## Llama Guard for GLUE MRPC task
+
+Llama Guard can be used for text classification task. The *transformers* library will change the head of the model for you during fine-tuning or inference. You can use same general command as for BERT, except you need to add `--add_pad_token=True`, because Llama based models don't have `pad_token` in their model and tokenizer configuration files. So, `--add_pad_token=True` will add `pad_token` to tokenizer and model's config as `eos_token` if it's not defined.
+
+### Fine-tuning with DeepSpeed
+
+Llama Guard can be fine-tuned with DeepSpeed, here is how you would do it on the text classification MRPC task using DeepSpeed with 8 HPUs:
+
+```bash
+python ../gaudi_spawn.py /
+    --world_size 8 --use_deepspeed run_glue.py \
+    --model_name_or_path meta-llama/LlamaGuard-7b \
```

Comment:
For 7b model, can you fit to 1 HPU? have you checked if the deepspeed has better perf than mpi?